config/routes.rb

```diff
@@ -3,5 +3,6 @@
   ActiveAdmin.routes(self)
   devise_for :users
   root "texts#index"
+  resources :texts
   resources :movies
```

Comment:
ルーティングは必要なものだけとしましょう。動画の方もついでに絞っていただけますか。

```suggestion
  resources :texts, only: [:index, :show]
  resources :movies, only: [:index]
```